app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -53,6 +60,20 @@
     private static final String TAG = "FeedInfoActivity";
 
     private Feed feed;
+    private final View.OnClickListener copyUrlToClipboard = new View.OnClickListener() {
```

Comment:
I don't think this should be moved. Listeners with a lot of code make the members section less readable.